### ISSUE TITLE
Add Passport to website

### ISSUE THREAD

**Issue by dzienisz** [naugtur/meetjs.pl#66]
Started as a interesting idea now it's a reality. meet.js Passport

**Comment by Eghizio** [comment]
Care to elaborate @dzienisz ? 😄

**Comment by dzienisz** [comment]
It's just and idea to make Meetjs Passport on the web. I have no idea yet how it should look like

**Comment by Eghizio** [comment]
## How about creating a PWA on subdomain.

The app would collect some `stamps for attending Meet.js events`.

Maybe `broadcasting some events via notifications` (**Web Push** & **Native Push**) to _increase the engagement_.

`Achievements` for reaching certain milestones, for example:
- _1st_ event
- 3 events in a _row_
- 5 _events_
- 1 year of _attendance_
- 2nd _city_ unlocked
- All _cities_ conquered

Some `quests/challenges` for gamification:
- Take a photo with 10 _attendees_
- Take a photo with 3 _organizers_
- Take a photo with 3 _speakers_
- Share event with 5 _people_

And so on and so on 😄 

Stamps could be in a form of a `hash/link/QR code` only valid during the events duration or some time threshold. Not sure about the online attendees.

We could have some `highscores board` to increase the gamification - or even include some `simple game` (Snake or something) to grind in app to introduce some competition 😏 

Stamps, scores or achievements could grant some `discounts` to paid events or grant some `coupons for drinks` during the event, or `meet.js merch` - or Summit discount or any partnering event/conference discounts and so on.

The ideas are quite limitless (just like AWS or Vercel billing) 😎

**Comment by dzienisz** [comment]
I like this idea!

**Comment by Eghizio** [comment]
Yeah, tho it is a big project idea. But might be something interesting for new people from the communities across all of the cities to join the project 😉

**Comment by dzienisz** [comment]
Like in RUBY community

- jest nfc i kod qr, więc łatwo się rejestrujesz jak już jesteś na evencie i ci się do rubyevents dodaje że byłeś
- pieczątkują na eventach żebyś sobie pieczątki zbierał

![Image](https://github.com/user-attachments/assets/ccdd00b8-40ae-4ef2-ba9f-87e106251aa2)

<img width="477" height="328" alt="Image" src="https://github.com/user-attachments/assets/67851c99-1734-442b-98cf-76edfc4c0873" />
